File: validation-numbers-jumia/src/main/java/com/example/jumiatask/entity/Customer.java
package com.example.jumiatask.entity;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "customer")
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Customer {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;
    private String name;
    private String phone;
    @Transient
    private String state;
    @Transient
    private String country;

    public Customer(int id , String name , String phone){
        this.id = id;
        this.name = name;
        this.phone = phone;
    }
}

File: validationNumbersOfJumia/src/app/model/customer.ts
export class Customer {

    id: number;
    name: String;
    phone: String;
    state: String;
    country: String;

    constructor(public c_Id: number, public c_Name: String, public c_Phone: String, public c_State: String, public c_Country: String)
     { 
         this.id=c_Id;
         this.name = c_Name;
         this.phone = c_Phone;
         this.state = c_State;
         this.country = c_Country;
     }


}
